### ISSUE TITLE
[1.8.2] Add fields to computers/printers/phones models

### ISSUE THREAD

**Issue by Kiks67** [pluginsGLPI/fields#287]
Hello,

When I add a new field to "computer models", the field doesn't appear in the form.
But the field seems to exist (see the screenshot # 2).

![image](https://user-images.githubusercontent.com/44313220/47343522-8b03d700-d6a6-11e8-86dc-1ac51c4daf41.png)

![image](https://user-images.githubusercontent.com/44313220/47343557-a4a51e80-d6a6-11e8-8f3d-8e31d987592e.png)

Same problems found in : 
- others models (monitors, phones, etc.)
- operating systems

Configuration : 
GLPI 9.3.1
Fields 1.8.2

**Comment by Du-St** [comment]
Hello,

I am facing the same problem. I have associated new block to some Dropdowns (models or types, i.e. Computer model, Monitor model, Model of previously created Generic object, ...), but no custom field has appeared.

GLPI 9.4.4 (RPM package on Fedora 30)
Fields 1.10.2

Additionally I made some investigation. The issue appears only on models without support of the Entity. This means there is no entities_id column in the appropriate dropdown table (i.e. glpi_computermodels).

I am attaching the ugly "patch". This patch disables some checks connected to the Entity in the Fields plugin. After applying the patch the custom fields appear on the dropdown form pages and it is also possible to select and successfully save the selected values.

I hope somebody will create the patch to solve this issue in the right way.

[glpi_fields_models.txt](https://github.com/pluginsGLPI/fields/files/3888638/glpi_fields_models.txt)
